.github/workflows/appstore-build-publish.yml

```diff
@@ -0,0 +1,120 @@
+# This workflow is provided via the organization template repository
+#
+# https://github.com/nextcloud/.github
+# https://docs.github.com/en/actions/learn-github-actions/sharing-workflows-with-your-organization
+
+name: Build and publish app release
+
+on:
+  release:
+    types: [published]
+
+env:
+  PHP_VERSION: 7.4
+
+jobs:
+  build_and_publish:
+    runs-on: ubuntu-latest
+
+    # Only allowed to be run on nextcloud-releases repositories
+    if: ${{ github.repository_owner == 'nextcloud-releases' }}
+
+    steps:
+      - name: Check actor permission
+        uses: skjnldsv/check-actor-permission@v2
```

Comment:
Can you move this to a NC owned repo?

Reply:
Not sure it's required. I'm the creator and it's widely available for github.
I'm the only one maintaining it :shrug: